{
  "file": "core/src/result.rs",
  "name": "<core::result::Result<vec::Vec<u8>, collections::TryReserveError> as core::ops::FromResidual<core::result::Result<core::convert::Infallible, collections::TryReserveError>>>::from_residual",
  "hash": "167388382150536567101328838122241772180",
  "hash_direct": "152353618162941918699469610952627663618",
  "src": "fn from_residual(residual: Result<convert::Infallible, E>) -> Self {\n        match residual {\n            Err(e) => Err(From::from(e)),\n        }\n    }",
  "callees": [
    "112317438206578259322527982339728828336"
  ]
}
